dds/DCPS/Service_Participant.cpp

```diff
@@ -163,6 +164,9 @@ static bool got_log_verbose = false;
 static bool got_default_address = false;
 static bool got_bidir_giop = false;
 static bool got_thread_status_interval = false;
+static bool got_relay_thread_key = false;
+static bool got_relay_thread_hwm = false;
```

Comment:
This should not appear in OpenDDS proper.  These are solely for the benefit of the rtps relay so it should be handled entirely in the relay.

Reply:
While I can refactor computation of thread load and saturation into relay-specific code, I think the relay_thread_key attribute needs to stay because that identifies the thread who's saturation status determines if message senders are held back for a while.